core/controllers/collection_viewer_test.py

```diff
@@ -129,60 +129,58 @@ def test_welcome_collection(self):
         self.assertEqual(
             collection_dict['title'], 'Introduction to Collections in Oppia')
 
-        # Verify there are 5 explorations in this collection, the initial
+        # Verify there are 3 explorations in this collection, the initial
```

Comment:
4?

Reply:
Ugh, I had updated this once and missed it the second time. :( Done.